src/connection.ts

```diff
@@ -0,0 +1,66 @@
+import { Connection }  from '@libp2p/interface-connection'
+import {ConnectionStat}from '@libp2p/interface-connection'
```

Comment:
Some inconsistent formatting.  Try installing `prettier` and add to `format on save` in you editor.

Reply:
The aim there was aligning `from`. 

Changed my settings as recommended, though. Should be prettier in the future.